source/includes/indexes/single-field.scala

```diff
@@ -42,6 +42,16 @@ object SingleFieldIndex {
                             (e: Throwable) => println(s"There was an error: $e"))
     // end-check-single-index
 
+    // start-drop-single-index
```

Comment:
S: if you create a separate index usage examples file, I'd move these two examples there